src/packages/core/property-editor/uis/tree-picker/property-editor-ui-tree-picker.element.ts

```diff
@@ -36,12 +39,19 @@ export class UmbPropertyEditorUITreePickerElement extends UmbLitElement implemen
 	@state()
 	ignoreUserStartNodes?: boolean;
 
+	#dynamicRoot?: UmbTreePickerSource['dynamicRoot'] | undefined;
```

Comment:
You don't need to specify undefined explicitly as the `?` already implies this.

```suggestion
	#dynamicRoot?: UmbTreePickerSource['dynamicRoot'];
```